api/plot/utils.py

```diff
@@ -213,6 +215,8 @@ def table(
         self,
         dataArray: OptionalLists,
         cellWidth: int,
+        display_style: Union[TableDisplayType, None] = None,
```

Comment:
Switch to using `|` instead of `Union` as we're using Python `3.10`

Reply:
Or better to have `Optional[TableDisplayType] = None` here?

Reply:
I think `TableDisplayType | None` is preferred as you don't need the additional import